pool/web/handler/contact_user_profile.ml

```diff
@@ -147,6 +161,152 @@ let update_password req =
   result |> HttpUtils.extract_happy_path_with_actions ~src req
 ;;
 
+let update_phone_number req =
+  let%lwt urlencoded = Sihl.Web.Request.to_urlencoded req in
+  let result
+    ({ Pool_context.database_label; language; query_language; _ } as context)
+    =
+    let open Utils.Lwt_result.Infix in
+    let tags = tags req in
+    Utils.Lwt_result.map_error (fun msg ->
+      HttpUtils.(
+        msg, "/user/contact-information", [ urlencoded_to_flash urlencoded ]))
```

Comment:
`"/user/contact-information"` is used in many places bellow, extract to a variable